ui/src/views/settings/SettingsMailboxes.vue

```diff
@@ -98,6 +98,58 @@
             </cv-row>
           </cv-grid>
         </cv-tile>
+        <cv-tile light>
+          <cv-grid fullWidth class="no-padding">
+            <cv-row>
+              <cv-column>
+                <h4 class="mg-bottom-md">
+                  {{ $t("settings_mailboxes.sharedseen") }}
+                  <cv-tooltip
+                    alignment="start"
+                    direction="bottom"
+                    :tip="$t('settings_mailboxes.sharedseen_explanation_tooltips')"
+                    class="info mg-bottom"
+                  >
+                  </cv-tooltip>
+                </h4>
+                <cv-skeleton-text
+                  v-if="loading.getMailboxSettings"
+                  heading
+                  paragraph
+                  :line-count="4"
+                  width="80%"
+                ></cv-skeleton-text>
+                <cv-form v-else @submit.prevent="setMailboxSettings">
+                  <NsToggle
+                    value="sharedSeenValue"
+                    :form-item="true"
+                    v-model="sharedseen.enabled"
+                    :disabled="
+                      loading.getMailboxSettings || loading.setMailboxSettings
+                    "
+                    :class="toggle - without - label"
```

Comment:
```suggestion
                    :class="toggle-without-label"
```
I guess this is a typo